debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorTask.java

```diff
@@ -201,11 +203,13 @@ public void stop() {
         private final BlockingQueue<SourceRecord> records;
         private final BooleanSupplier isRunning;
         private final Consumer<List<SourceRecord>> batchConsumer;
+        private final long returnIntervalMs;
```

Comment:
It's the age of Java 8, let's use more expressive types such as `java.time.Duration`.

Reply:
Does it really makes sense? Right now it would conver to this sequence of events
 - read long form Configuration and convert it to Duration
 - convert Duration into long and pass it to Timer

Reply:
I'd prefer if we globally moved to `Duration` (and other time types as needed), so let's start doing it in new code. I.e. `Timer` should receive a `Duration` to.  We can adjust existing usages as we go.